vmware_exporter/vmware_exporter.py

```diff
@@ -207,8 +231,50 @@ def collect(self, target=None, section='default'):
         for metricname, metric in metrics.items():
             yield metric
 
+    def _collect_subsystems(self, section, valid_subsystems):
+        """
+          Return the list of subsystems to collect - everything by default, a
+          subset if the config section has collect_only specified
+        """
+        collect_subsystems = []
+
+        if not self.config[section].get('collect_only'):
+            collect_subsystems = valid_subsystems
+        else:
+            for subsystem in self.config[section].get('collect_only'):
+                if subsystem in valid_subsystems:
+                    collect_subsystems.append(subsystem)
+                else:
+                    print("invalid subsystem specified in collect_only: " + str(subsystem))
+
+            if not collect_subsystems:
+                print("no valid subystems specified in collect_only, collecting everything")
+                collect_subsystems = valid_subsystems
+
+        return collect_subsystems
+
+
+    def _collect_subsystems(self, section, valid_subsystems):
```

Comment:
I think there is a mistake here (duplicates)

Reply:
def _collect_subsystems
228 -248 string in fork.. strange merge.

Reply:
Probably a bad merge by me, let me take a peek

Reply:
Yikes, yeah, I'm gonna say bad merge;  let me clean that up